src/modules/sdk/objections/service.ts

```diff
@@ -154,13 +155,22 @@ export const downloadAttachment = async (downloadApiUrl: string,
       Authorization: "Bearer " + token,
     },
     proxy: false,
+    validateStatus: (status) => {
+      return ((status >= 200 && status < 300) || status === 401);
```

Comment:
A 401 should throw an ApiError anyway from makeApiCall

default is 
// `validateStatus` defines whether to resolve or reject the promise for a given
  // HTTP response status code. If `validateStatus` returns `true` (or is set to `null`
  // or `undefined`), the promise will be resolved; otherwise, the promise will be
  // rejected.
  validateStatus: function (status) {
    return status >= 200 && status < 300; // default
  },

plus make sure the other codes get thrown as well:
  UNAUTHORIZED = 401,
  FORBIDDEN = 403,
  NOT_FOUND = 404,
  UNSUPPORTED_MEDIA_TYPE = 415,

Reply:
It does throw an api error for 401 in the makeApiCall but the api error was never making it to the document download controller for it to interperet and render the correct page and I couldn't figure out where it was going

Reply:
hold on looking at the original error again, i may have a better solution